csharp-101/Intro To Machine Learning.ipynb

```diff
@@ -0,0 +1,320 @@
+{
```

Comment:
let's move this under machine-learning folder (new folder but we'll all use it).

Reply:
done!